packages/story-editor/src/app/media/utils/useDetectBaseColor.js

```diff
@@ -86,7 +86,9 @@ function useDetectBaseColor({ updateMediaElement }) {
       if (type === 'image') {
         imageSrc = getSmallestUrlForWidth(0, resource);
       } else if (!isExternal) {
-        const posterResource = await getPosterMediaById(id);
+        const posterResource = getPosterMediaById
+          ? await getPosterMediaById(id)
+          : '';
```

Comment:
```suggestion
          : null;
```

Seems more intuitive to use `null` instead of an empty string to indicate that this value is absent.